easybuild/easyconfigs/i/i-PI/i-PI-1.0-20160213-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name  ='i-PI'
+version = '1.0-20160213'
+commit = '2a09a6d'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/i-pi/i-pi'
+description = """A Python wrapper for (ab initio) (path integrals) molecular dynamics"""
```

Comment:
it wraps several things, but does so at runtime; including something specific as a dependency would artificially limit it to one particular things